scripts/correlateTBN.py

```diff
@@ -128,11 +128,16 @@ def main(args):
                 station = metabundle.get_station(args.metadata, apply_sdm=True)
             except:
                 station = metabundleADP.get_station(args.metadata, apply_sdm=True)
+    elif args.lwasv:
+        station = stations.lwasv
     else:
         station = stations.lwa1
     antennas = station.antennas
-    
-    idf = LWA1DataFile(filename)
+
+    if args.lwasv:
+        idf = LWASVDataFile(filename)
+    else:
+        idf = LWA1DataFile(filename)
```

Comment:
Since `LWADataFile` is fairly generic there should be some catch to make sure that the data access class returned by this function is of type `lsl.reader.ldp.TBNFile`.  I don't remember I had done that in the script.

Reply:
OK I added a `if not isinstance(idf, TBNFile):` check.

Thanks for pointing me towards `LWADataFile`!